docs/README.md

```diff
@@ -1,7 +1,7 @@
 ## Docs
 
-- [See an overview of BLS Wallet & how the components work together](./docs/system_overview.md)
-- [Use BLS Wallet in a browser/NodeJS/Deno app](./docs/use_bls_wallet_clients.md)
+- [See an overview of BLS Wallet & how the components work together](./system_overview.md)
+- [Use BLS Wallet in a browser/NodeJS/Deno app](./use_bls_wallet_clients.md)
 - Setup the BLS Wallet components for:
-  - [Local develeopment](./docs/local_development.md)
-  - [Remote development](./docs/remote_development.md)
\ No newline at end of file
+  - [Local develeopment](./local_development.md)
+  - [Remote development](./remote_development.md)
```

Comment:
I noticed this section was duplicated from `./README.md` (or vice versa).
Do we need a root level readme, and this partial copy in docs/readme?

Reply:
This is a duplicate of what we have in the [top level README](https://github.com/web3well/bls-wallet/blob/main/README.md#getting-started) to give context over what is included in current docs. I just messed up my relative pathing when I copy pasta'd.

Open to removing it from the `docs` README if you think it is redundant.